docker-images/kafka/scripts/kafka_internal_certs_import.sh

```diff
@@ -1,5 +1,4 @@
 #!/bin/bash
 
```

Comment:
I notice now the very similar code between the `kafka_clients_certs_import.sh` and `kafka_internal_certs_import.sh`. I wonder if we should factor out a common script that defines bash functions for those things, and source that script in the the `kafka_clients_certs_import.sh` and `kafka_internal_certs_import.sh`?

Reply:
@tombentley I did some updates to the scripts to make sure we reuse the code. I would appreciate if you could have a look at it again.